### PR TITLE
Wazo 1182 remove dahdi dependency

### PR DESCRIPTION
**Pull request by pc-m**
Depends-On: https://github.com/wazo-platform/wazo-confgend/pull/17

### CODE REVIEW

File: debian/wazo-asterisk-config.postinst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# postinst script
+#
+# see: dh_installdeb(1)
+
+set -e
+
+case "$1" in
+    configure)
+        if [ -d /run/systemd/system ]; then
+            systemctl --system daemon-reload >/dev/null || true
```
Comment (fblackburn1) [inline debian/wazo-asterisk-config.postinst]:
```suggestion
            systemctl daemon-reload >/dev/null || true
```
the --system is implied by default

File: debian/wazo-asterisk-config.postrm [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#! /bin/sh
+
+set -e
+
+if [ "$1" = "remove" ]; then
+    if [ -d /run/systemd/system ]; then
+        systemctl --system daemon-reload >/dev/null || true
```
Comment (fblackburn1) [inline debian/wazo-asterisk-config.postrm]: see above

File: debian/wazo-asterisk-config.postinst [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#! /bin/sh
+# postinst script
+#
+# see: dh_installdeb(1)
+
+set -e
+
+case "$1" in
+    configure)
+        if [ -d /run/systemd/system ]; then
```
Comment (fblackburn1) [inline debian/wazo-asterisk-config.postinst]: This check is redundant with the `|| true`
we should remove it, or remove `|| true`

File: debian/wazo-asterisk-config.postrm [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#! /bin/sh
+
+set -e
+
+if [ "$1" = "remove" ]; then
+    if [ -d /run/systemd/system ]; then
```
Comment (fblackburn1) [inline debian/wazo-asterisk-config.postrm]: see above

**Comment by pc-m** [comment]
This is a comment I got from someone else on another PR
```
dh_systemd_start uses this:

if [ -d /run/systemd/system ]; then
    systemctl --system daemon-reload >/dev/null || true
fi
```

**Comment by fblackburn1** [comment]
> This is a comment I got from someone else on another PR
> 
Which?
for what I see, db_systemd_start is a perl script that doesn't use what you write

**Comment by pc-m** [comment]
> > This is a comment I got from someone else on another PR
> 
> Which?
https://github.com/wazo-platform/wazo-res-stasis-amqp/pull/13/files/c5307299eb0090dcc1c27e055d1d664e71171c22#diff-ec5b912e2373f138274fb3cf2033eda2
> for what I see, db_systemd_start is a perl script that doesn't use what you write
The perl script inserts this code `/usr/share/debhelper/autoscripts/postinst-systemd-restartnostart`

**Comment by fblackburn1** [comment]
Interesting, thanks for the infos

**Pull request merged**